src/build_system.jl

```diff
@@ -1,12 +1,16 @@
-function build_system(name::String; kwargs...)
-    #TODO: add check for supported kwargs
-    add_forecasts = get(kwargs, :add_forecasts, true)
-    add_reserves = get(kwargs, :add_reserves, false)
+function get_system_library(; kwargs...)
     if check_for_serialized_descriptor()
-        system_library = deserialize()
+        return deserialize()
     else
-        system_library = parse_system_library()
+        return parse_system_library()
     end
+end
+
+function build_system(name::String, print_stat::Bool=false; kwargs...)
```

Comment:
A valuable addition would be to allow passing time_series_read_only to the created system. It will avoid copying the time series .h5 file.